src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,3 +1,31 @@
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      {' Posted by  '}
```

Comment:
This test should be moved to `UserInfo` component, cause in case user === null, this text will look strange